ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPSystem.swift

```diff
@@ -11,4 +11,13 @@ import Foundation
 struct KMPSystem: Codable {
   public var keymanDeveloperVersion: String?
   public var fileVersion: String
+
+  init() {
+    keymanDeveloperVersion = nil
+
+    // Serves as a simple-enough default.  Not sure what the actual
+    // rules that compiled packages follow for this are... but we
+    // don't actually use this value within KeymanEngine, anyway.
+    fileVersion = "1.1.0"
```

Comment:
The file version for .kmp files is always `7.0`. That was the version of Keyman where they were introduced and the format hasn't changed since then (well, we introduced kmp.json but that's a separate story).

See https://github.com/keymanapp/keyman/blob/master/windows/src/global/delphi/general/PackageInfo.pas#L547

Reply:
Except when it isn't.

From `nrc.en.mtnt` v0.1.4's `kmp.json`:
```
  "system": {
    "keymanDeveloperVersion": "12.0.1500.0",
    "fileVersion": "12.0"
  },
```

Reply:
Ah yes. If it's a package that just includes kmp.json, then it's `12.0` -- so far that's only lexical model packages. Sorry, forgot that the model compiler was doing that.

Reply:
This value is now autogenerated depending on which resource type is being wrapped.  Keyboards => `"7.0"`, lexical models => `"12.0"`.

Thanks again for clarifying.